vendor/LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Don't remove other's license.

Reply:
That license is same as ours (verified with diff(1))